### DISCUSSION TITLE
Implementation of verifyMessage

### DISCUSSION THREAD

**Discussion by saklani** [wevm/viem#185] [Idea / Feature Request] [answered]
Hi! I am using viem for a project and I need verifyMessage, I'd like to help implement it if its not being currently worked on.

**Comment by jxom** [comment, marked as answer]
It’s on the Roadmap! Open to contributions. Could be worth using [this library](https://github.com/paulmillr/noble-secp256k1) for implementation.

https://github.com/orgs/wagmi-dev/projects/3?pane=issue&itemId=21305038

**Reply by saklani** [reply]
Alright cool, then I'll start implementing it.
